daprdocs/content/en/reference/components-reference/supported-bindings/_index.md

```diff
@@ -17,7 +17,7 @@ Table captions:
 > `Status`: [Component certification]({{<ref "certification-lifecycle.md">}}) status
   - [Alpha]({{<ref "certification-lifecycle.md#alpha">}})
   - [Beta]({{<ref "certification-lifecycle.md#beta">}})
-  - [GA]({{<ref "certification-lifecycle.md#general-availability-ga">}})
+  - [Stable]({{<ref "certification-lifecycle.md#stable">}})
```

Comment:
None of these stable changes should be in this PR, correct? These have already been applied by @artursouza. Please remove them